bundled/tool/server.py

```diff
@@ -36,14 +36,16 @@ def update_sys_path(path_to_add: str, strategy: str) -> None:
 # **********************************************************
 # pylint: disable=wrong-import-position,import-error
 import jsonrpc
+import lsprotocol.types as lsp
```

Comment:
@ischaojie You may have to replace all `lsp.Trace.` in that file with `lsp.TraceValues.`

Reply:
Oh, sorry, I forgot it. I will fix it.